spec/embedded_sass.proto

```diff
@@ -1060,3 +1067,17 @@ enum CalculationOperator {
   // The division operator.
   DIVIDE = 3;
 }
+
+// The built-in Node.js Package Importer, which is a Package Importer that
+// resolves using the standards and conventions of the Node.js ecosystem. It
+// enables a `pkg:` URL scheme for usage with `@use` that directs an
+// implementation to resolve a URL within a dependency. 
+message NodePackageImporter {
+  // The path to associate with the Node Package Importer, with semantics
+  // identical to the [entryPointPath option] in the JavaScript API. It can
+  // either be an absolute path, or a relative path that will be resolved
+  // relative to the current working directory.
```

Comment:
We should enforce this to be only absolute path. Relative path should be resolve by the embedded host.

See this comment for background: https://github.com/sass/embedded-host-node/pull/261#issuecomment-1877725010

Reply:
Good catch, updated in https://github.com/sass/sass/pull/3728/commits/6f7bf999ceb65edb0f36923b1a32e19a178b984d